src/Components/Browser.JS/src/Boot.Server.ts

```diff
@@ -18,7 +26,9 @@ async function boot() {
     return loadEmbeddedResourcesAsync(bootConfig);
   });
 
-  const initialConnection = await initializeConnection(circuitHandlers);
+  // pass options.configureSignalR to configure the signalR.HubConnectionBuilder
+  const configureSignalR = (options || {}).configureSignalR
```

Comment:
Nit: missing semicolon

Reply:
Other nit: `(options && options.configureSignalR) || null` is fractionally more idiomatic, and avoids instantiating an empty object that will then just be discarded.